remove extra ```bash annotations

stuartleeks opened the pull request:
The "reinstall draft" section has a trailing ```bash in the output (and a trailing ``` in the next section). This change is intended to remove them.

azurecla commented:
Hi __@Drewm3__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

stuartleeks committed:
remove extra ```bash annotations

The "reinstall draft" section has a trailing ```bash in the output (and a trailing ``` in the next section). This change is intended to remove them.

GitHubber17 committed:
Update container-service-draft-up.md

GitHubber17 committed:
Update container-service-draft-up.md
